examples/09-validate-validations.js

```diff
@@ -0,0 +1,11 @@
+// Example of setting the `validations` of a content type.
+module.exports = function (migration) {
+  const food = migration.editContentType('food');
+
+  food.createField('boo')
+    .type('Symbol')
+    .name('what it tastes like')
+    .validations([{ unique: true }]);
+
+  food.displayField('taste');
+};
```

Comment:
maybe we should have an integration test using this example?

Reply:
Oh, true, there's actually a ticket for that 😅